### DISCUSSION TITLE
Contributing to ABP Framework

### DISCUSSION THREAD

**Discussion by tinonetic** [abpframework/abp#11531] [General] [answered]
Hi there,

I absolutely enjoy using the ABP Framework.  

We have drastically improved our productivity and quality of output since we adopted it from the ASP Net Boiler Plate days.

I would love to contribute to the project. 

I have zero experience contributing to opensource, but I looked at the [help wanted tag.](https://github.com/abpframework/abp/labels/help%20wanted) and all I see is an old(2018) NHibernate request.

So does this mean there are no current contribution requirements? 
Do you source your own contributors?
Would I have to wait to personally identify a potential gap or issue, so that I can send in a pull request?
Any simple requirements for first-time OSS contributors?

Kind Regards,
Tino

**Comment by maliming** [comment, marked as answer]
hi

https://docs.abp.io/en/abp/latest/Contribution/Index
